src/outbackcdx/Capture.java

```diff
@@ -156,21 +168,40 @@ public void encodeValue(ByteBuffer bb) {
     }
 
     /**
-     * Format as a CDX11 line.
+     * Format as a CDX11 line, or CDX14, depending on what fields are present.
      */
     public String toString() {
         StringBuilder out = new StringBuilder();
-        out.append(urlkey).append(' ');
-        out.append(Long.toString(timestamp)).append(' ');
-        out.append(original).append(' ');
-        out.append(mimetype).append(' ');
-        out.append(Integer.toString(status)).append(' ');
-        out.append(digest).append(' ');
-        out.append(redirecturl).append(' ');
-        out.append(robotflags).append(' ');
-        out.append(Long.toString(length)).append(' ');
-        out.append(compressedoffset).append(' ');
+        out.append(urlkey).append(" ");
+        out.append(Long.toString(timestamp)).append(" ");
+        out.append(original).append(" ");
+        out.append(mimetype).append(" ");
+        out.append(Integer.toString(status)).append(" ");
+        out.append(digest).append(" ");
+        out.append(redirecturl).append(" ");
+        out.append(robotflags).append(" ");
+        out.append(Long.toString(length)).append(" ");
+        out.append(compressedoffset).append(" ");
```

Comment:
What's the reason for changing these?

Reply:
Consistency with other strings in the file, I forgot I'd changed these, my apologies, I can remove them if you wish, but may be advantageous to keep them in for consistency.